src/Aws/Sns/MessageValidator/MessageValidator.php

```diff
@@ -34,12 +34,20 @@ class MessageValidator
      */
     protected $client;
 
+    /** @var string */
+    private $hostPattern;
+    private static $defaultHostPattern
+        = '/^sns\.[a-zA-Z0-9\-]{3,}\.amazonaws\.com(\.cn)?$/';
+
     /**
      * Constructs the Message Validator object and ensures that openssl is installed
```

Comment:
Can you add a line here that documents the host pattern and what the default value allows?